magento-research/pwa-studio
# [doc] add product diagrams

## Description

* Add the product diagrams to appropriate topics in the doc site.
* Update topic content to reflect current state of the project.
* Clean up any linting errors in the modified topics.

## Related Issue

Closes #1347 

## How have YOU tested this?

See verification steps

## Verification Steps

1. Navigate to the `pwa-devdocs` directory
2. Run linter on the technology overview file:
    ```sh
   npm run lint src/technologies/overview/index.md
   ```
3. Verify no linting issues associated with this file.
4. Run linter on the tools and library file:
    ```sh
    npm run lint src/technologies/tools-libraries/index.md
    ```
5. Verify no linting issues associated with this file.
6. Build and run the preview server: `npm run develop`
7. Navigate to `/technologies/overview/`
8. Verify the developer overview image is on the page.
9. Navigate to `/technologies/tools-libraries/`
10. Verify the tools and technology image is on the page.


## Checklist:
